config/rbac/role.yaml

```diff
@@ -101,10 +101,6 @@ rules:
   verbs:
   - delete
   - get
-  - list
-  - patch
-  - update
-  - watch
```

Comment:
Can you also remove `list` and `watch` from the `virtualmachines` entry that's slightly below this one